wsgi.py

```diff
@@ -24,24 +24,25 @@
 
 import daiquiri
 
-from flask import Flask
-from flask import jsonify
-from flask import request
+from flask import Flask, Response, jsonify, request
 from flask.helpers import make_response
 
 import requests
 
 from prometheus_client import CONTENT_TYPE_LATEST
-from prometheus_client import Counter
-from prometheus_client import Histogram
-from prometheus_client import core
-from prometheus_client import generate_latest
+from prometheus_client import Counter, Histogram
+from prometheus_client import core, generate_latest
```

Comment:
seems to be good for such a small file, but in general I guess you would move all prometheus stuff into its own package to not to pollute the namespace here. Eg. `CONTENT_TYPE_LATEST`
Or just `import prometheus` and reference with a FQN like `prometheus.CONTENT_TYPE_LATEST`